### ISSUE TITLE
Add support for Microsoft Video Indexer and/or Google object tracking

### ISSUE THREAD

**Issue by tyarkoni** [tyarkoni/pliers#329]
Tracking objects/people in videos is a high priority. Of the major available services, AWS Rekognition requires upload to an S3 bucket, so we can defer that. Azure's Video Indexer looks like it should be fairly easy to add. Google now has a beta Object Tracking mode in Video Intelligence. @qmac, is there a specific reason you never implemented Object Tracking, or was it just not available at the time and/or you didn't get to it before you left? If the latter, I'll probably add it myself, but just wanted to make sure you didn't already look into it and decide it was unworkable for some reason.

**Comment by qmac** [comment]
Yeah all I remember was it being complicated to do. IIRC for the Microsoft Face tracking, you had to get the IDs for an original set of faces and then query for those in subsequent requests. Not sure if all the tracking APIs operate that way, its all kind of hazy at this point.
